book/validation.rst

```diff
@@ -329,7 +329,7 @@ Some constraints, like :doc:`NotBlank </reference/constraints/NotBlank>`,
 are simple whereas others, like the :doc:`Choice </reference/constraints/Choice>`
 constraint, have several configuration options available. Suppose that the
 ``Author`` class has another property called ``gender`` that can be set to either
-"male" or "female":
+"male", "female", or "other":
```

Comment:
Can you please remove the comma in front of "or"? We don't use serial commas in the documentation.